src/kafkaaggregator/config.py

```diff
@@ -71,27 +73,60 @@ class Configuration:
     workload of the application.
     """
 
-    src_topic: str = os.getenv("SOURCE_TOPIC", "kafkaaggregator-src-topic")
+    source_topic_name: str = os.getenv(
+        "SOURCE_TOPIC", "kafkaaggregator-example"
+    )
     """Name of the source topic used in the kafkaaggregator example."""
 
-    agg_topic: str = os.getenv(
-        "AGGREGATION_TOPIC", "kafkaaggregator-agg-topic"
+    topic_regex: str = os.getenv("TOPIC_REGEX", "^kafkaaggregator-example$")
+    """Regex to select topics to aggregate."""
+
+    topic_blacklist: List[str] = field(default_factory=list)
```

Comment:
I think we're trying to avoid the term "blacklist". Maybe "excluded_topics" ?

Reply:
Agreed. It does sound bad. Kafka uses balcklist/whitelist quite a lot and I was motivated by that.